spec/oncocore_vs.txt

```diff
@@ -3,10 +3,1303 @@ Namespace:	oncocore
 
 CodeSystem:     SCT = http://snomed.info/sct
 CodeSystem:	    MTH = http://ncimeta.nci.nih.gov
+CodeSystem:     ICD10CM = https://www.cdc.gov/nchs/icd/icd10cm.htm
```

Comment:
incorrect link for icd10cm.  reference HL7 documentation on [Using ICD codes with FHIR](http://hl7.org/fhir/icd.html).